config/sync/structure_sync.data.yml

```diff
@@ -215,8 +231,8 @@ menus:
     title: 'Information for...'
     parent: null
     uri: 'entity:node/2'
-    link_title: null
-    description: null
+    link_title: ''
+    description: 'Our quick start guides to help find  the right information for you.'
```

Comment:
Looks like a double space between 'find' and 'the'

Reply:
That's what happens when you copy and paste from Photoshop... I'll tidy it up in the next pull request